gitbook-docs/README.md

```diff
@@ -10,20 +10,18 @@ a computational infrastructure, scale it, and specialize as a proof producer.
 
 `=nil;` understands that generating proofs is a task that should be outsourced to such
 a specialized provider.
-This is exactly why we are building the Proof Market — a marketplace where everyone can request a zkProof,
+This is exactly why we are building Proof Market — a marketplace where everyone can request a zkProof,
 and a network of specialized producers will respond to such requests.
 
-We see the Proof Market as the place where proof requesters and proof producers will meet
+We see Proof Market as the place where proof requesters and proof producers will meet
```

Comment:
Let's make it sound a bit more confident :)
```suggestion
We see Proof Market as the place where proof requesters and proof producers meet
```

Reply:
Fixed